.circleci/config.yml

```
@@ -597,6 +609,14 @@ workflows:
               only: main
           requires:
             - Check Experimenter (Main)
+      - cirrus_deploy:
+          name: Deploy Cirrus
+          filters:
+            branches:
+              ignore:
```

I'm thinking this may have been left in for testing but we'll def want this to be `only` before merging.
```suggestion
              only:
```

---

Awesome just wanted to double check!